package.json

```diff
@@ -4,7 +4,8 @@
   "description": "This is an attempt to polyfill the [Text Fragments](https://wicg.github.io/scroll-to-text-fragment/) feature for browsers that don't support it natively.",
   "main": "text-fragments.js",
   "dependencies": {
-    "prettier": "^2.0.5"
+    "prettier": "^2.0.5",
+    "terser": "^4.8.0"
```

Comment:
Is this necessary? We run terser as a plugin on rollup.

Reply:
I didn't see rollup-plugin-terser, removing this because it's not necessary.